src/js/components/Text/StyledText.js

```diff
@@ -74,6 +74,7 @@ const weightStyle = css`
 `;
 
 const StyledText = styled.span`
+  ${baseStyle}
```

Comment:
This doesn't seem right to me. I don't think the `baseStyle` shouldn't be brought in for every text component.